src/usr/local/www/vpn_ipsec_phase2.php

```diff
@@ -427,9 +441,11 @@
 
 $localid_help_tunnel  = "Local network component of this IPsec security association.";
 $localid_help_vti     = "Local point-to-point IPsec interface tunnel network address.";
+$localid_help_bypass  = "Local network that bypassed from IPsec processing.";
```

Comment:
Change `bypassed from` to `bypasses`.
Add ` for this IPsec connection` to the end since the bypass is setup as specific to this peer, not all peers.